flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.html

```diff
@@ -16,11 +16,21 @@
   ~ limitations under the License.
   -->
 
-<nz-tabset [nzSize]="'small'" [nzAnimated]="{ inkBar: true, tabPane: false }">
-  <nz-tab nzTitle="Root Exception" nzForceRender>
-    <flink-monaco-editor [value]="rootException"></flink-monaco-editor>
+<nz-tabset
+  [nzSize]="'small'"
+  [nzAnimated]="{ inkBar: true, tabPane: false }"
+  [(nzSelectedIndex)]="index"
+>
+  <nz-tab nzTitle="Root Exception">
+    <nz-code-editor
+      *ngIf="index === 0"
```

Comment:
Why do we need to keep track of the selected tab now?

Reply:
Well, switching the tab would not actually destroy the editor. I needed to manually destroy it through ngIf.

Reply:
I just come up with a better way to do this. I would use the lazy loading feature in nz-tabset:https://ng.ant.design/components/tabs/en#components-tabs-demo-lazy